prestodb/hdp2.6-hive/README.md

```diff
@@ -0,0 +1,13 @@
+# hdp2.5-hive [![][layers-badge]][layers-link] [![][version-badge]][dockerhub-link]
+           
+[layers-badge]: https://images.microbadger.com/badges/image/prestodb/hdp2.5-hive.svg
+[layers-link]: https://microbadger.com/images/prestodb/hdp2.5-hive
+[version-badge]: https://images.microbadger.com/badges/version/prestodb/hdp2.5-hive.svg
+[dockerhub-link]: https://hub.docker.com/r/prestodb/hdp2.5-hive
```

Comment:
Should the link content include "2.6" rather than "2.5"? (i didn't check other files!)

Reply:
done